www/assets/css/search.scss

```diff
@@ -91,7 +91,13 @@
         padding: 0 3rem;
         @include breakpoint(tablet) {
           padding: 0;
+          margin-left: -15px !important;
```

Comment:
Same as before; previously this was being applied through `row` class but since we want to use `mx-auto` for after `tablet`, so applying manually here